controllers/components/kserve/kserve_support.go

```diff
@@ -241,28 +294,3 @@ func hashConfigMap(cm *corev1.ConfigMap) (string, error) {
 
 	return base64.RawURLEncoding.EncodeToString(h), nil
 }
-
-func ownedViaFT(cli client.Client) handler.MapFunc {
```

Comment:
we probably need an action to remove FT ownership in case of an upgrade

Reply:
It seems that the FT owner references on the resources that it had previously created will be removed when the FTs are removed, so I think all we need to do here is remove the FTs themselves, WDYT?

Reply:
My understanding is that if the FT is removed, then all the owned object are removed as well, so we would probably to:

1. re-own existing resources
2. remove FTs

There is some [logic in the deploy action ](https://github.com/opendatahub-io/opendatahub-operator/blob/main/pkg/controller/actions/deploy/action_deploy.go#L137-L139), not sure if it is generically applicable to FTs

Reply:
> What I meant about the ownership being removed when the FT is removed, is that the Kserve CR is already there as an additional owner at this stage (actually as the main owner because it has the controller: true set, where the FT ref didn't) because this new iteration of the reconciler has deployed it now.
> So I think as long as we run the deploy action first, and then remove the FTs, in my understanding everything will already have been re-owned. Does that sound correct?

Ah yes, now I got it, so yes it should work. The only things is to be sure there is not such `controller: true` in the current ownership list, in such case I guess the deploy would fail (well, the `SetControllerReference` would)